src/Tree/OctTree.cpp

```diff
@@ -547,6 +550,8 @@ void OctTree<ndim,ParticleType,TreeCell>::StockTree
               if (partdata[i].v[k] > cell.vbox.max[k]) cell.vbox.max[k] = partdata[i].v[k];
               if (partdata[i].v[k] < cell.vbox.min[k]) cell.vbox.min[k] = partdata[i].v[k];
             }
+            cell.amin = min(cell.amin,
+                            sqrt(DotProduct(partdata[i].atree,partdata[i].atree,ndim)));
```

Comment:
We might be memory limited here, so I'm not sure what we'd be saving. I can change it, but perhaps we should leave it until we profile the tree properly?